Gordon360/Controllers/NewsController.cs

```diff
@@ -209,6 +209,23 @@ public ActionResult<StudentNewsViewModel> EditPosting(int newsID, [FromBody] Stu
             return Ok(result);
         }
 
+        /// <summary>
+        /// (Controller) Edits a news image in the database
+        /// </summary>
+        /// <param name="newsID">The id of the news item to edit</param>
+        /// <param name="newImageData">The new image string updates the image value</param>
+        /// <returns>The updated news item</returns>
+        /// <remarks>The news item must be authored by the user and must not be expired and must be unapproved</remarks>
+        [HttpPut]
+        [Route("{newsID}/image")]
+        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.NEWS)]
+        // Private route to authenticated users - authors of posting or admins
```

Comment:
You can remove this inline comment, since this fact has been documented in the XMLDoc comment above.